### DISCUSSION TITLE
Suggestion: Add visual countdown to break on taskbar icon

### DISCUSSION THREAD

**Discussion by thracyturnix** [tom-james-watson/breaktimer-app#255] [Ideas]
Thanks for this application. Its very cool, and is helping me to take regular breaks. 

I find myself often wondering how much time i have left before taking the next enforced break, and to some extent plan my workflow in accord with this. It would be super useful if the coffee cup icon had some kind of positive indication of how much work time is available before the next break session. 

I hope you will consider this idea. Thankyou.

**Comment by tom-james-watson** [comment]
There was some work started on this, but some more graphics work is needed to actually have them added to the app: https://github.com/tom-james-watson/breaktimer-app/pull/108
